pkg/agent/cniserver/server_windows.go

```diff
@@ -47,9 +47,20 @@ func (s *CNIServer) hostNetNsPath(netNS string) string {
 }
 
 // isInfraContainer return if a container is infra container according to the network namespace path.
```

Comment:
not introduced by this PR, but should be: isInfraContainer returns true if a container...

Reply:
Done.